### PR TITLE
tests/core/basic20plus: run for core24

### PR DESCRIPTION
**Pull request by Meulengracht**

### CODE REVIEW

File: tests/core/basic20plus/task.yaml [hunk 1]
```diff
@@ -32,23 +38,20 @@ execute: |
     snap changes | MATCH "Done.*Initialize system state"
 
     echo "Check that a simple shell snap"
-    if os.query is-core22; then
-        snap install --edge "test-snapd-sh-${base_snap}"
```
Comment (valentindavid) [inline tests/core/basic20plus/task.yaml:36]: Do you know why we used `--edge`?

Comment (Meulengracht) [reply tests/core/basic20plus/task.yaml:36]: This was before it was pushed to stable (I.e) before the core22 base reached stable

**Review by valentindavid — APPROVED** [review]

File: tests/core/basic20plus/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/core/basic20plus/task.yaml:36]: well, we agreed with store team to keep test snaps in edge and non listed. I think this is the main raeson

**Review by sergiocazzolato — APPROVED** [review]
Thanks

**Pull request merged**